webpack/scenes/Subscriptions/components/SubscriptionsTable/components/Table.js

```diff
@@ -27,7 +27,7 @@ const Table = ({
   const groupingController = {
     isCollapseable: ({ rowData }) =>
       // it is the first subscription in the group
-      rowData.id === groupedSubscriptions[rowData.product_id].subscriptions[0].id &&
+      rowData.collapsible &&
       // the group contains more then one subscription
       groupedSubscriptions[rowData.product_id].subscriptions.length > 1,
```

Comment:
the groupedSubscriptions check might be redundant now that we have the collapsible field on the row itself